### ISSUE TITLE
Job category: model distillation (proposed) — cross-subnet bridge to SN97 Distil

### ISSUE THREAD

**Issue by wallscaler** [cathedralai/cathedral#90]
**Depends on:** #89 (prompt-firing strategy)

## One-liner (for launch post)

> *Model distillation (proposed): agent reasoning traces → distilled specialist models, in conversation with SN97 Distil.*

## Why this card category

Three reasons:

1. **It maps to Const's "Faster" direction** (per 2026-05-11 DM thread). Distillation is the highest-leverage application of Cathedral's data moat — agent traces become training data for specialist models.
2. **SN97 Distil is a live partner subnet** that's already running distillation as a mining task. Bridging Cathedral cards into SN97's training pipeline turns our cards into their input data, and the resulting models become Cathedral assets.
3. **It's the platform thesis made concrete.** Cards aren't just briefs — they're data. The card-to-model pipeline is the proof.

## Card definition

```python
RegistryEntry(
    card_id="model-distillation",
    jurisdiction=Jurisdiction.OTHER,  # global / not jurisdiction-bound
    topic="Specialist model distillation: target identification, dataset assembly, distillation plans",
    required_source_classes=(
        SourceClass.SECONDARY_ANALYSIS,  # arXiv preprints
        SourceClass.OTHER,                # HuggingFace model cards, benchmark suites
    ),
    refresh_cadence_hours=168,  # weekly — model release cycles
)
```

## Source pool (initial seeding — Cathedral builds this)

- **HuggingFace model index** (trending, new releases, finetunes)
- **HuggingFace leaderboards** (Open LLM Leaderboard, BigCode, etc.)
- **arXiv preprints** in `cs.CL`, `cs.LG` distillation track
- **Papers with Code** benchmark results
- **lm-evaluation-harness** results for popular models
- **Anthropic, OpenAI, Google, Meta release announcements** (capability frontiers)
- **Bittensor SN97 Distil** validator pulls (cross-subnet — see below)
- **Distillation literature** (Hinton 2015, DistilBERT, TinyLLaMA, etc.)

## Templates (`task_templates`)

1. **"Distillation target identification"** — *"Given task class `<task_class>` and quality bar `<min_metric>` on benchmark `<benchmark>`, identify the most cost-effective distillation candidate. Return a Card with: title, summary, what_changed (recent model releases or papers since last refresh), teacher_model recommendation (name + size + benchmark scores), student_model_class recommendation (size + arch family), distillation_plan (data sources, expected compute, key hyperparams), citations (at least one arXiv + one benchmark result), confidence."*

2. **"Reasoning-trace dataset assembly"** — *"Given task class `<task_class>`, identify what kind of reasoning traces would best fine-tune a distilled student model. Return a Card with: title, summary, dataset_specification (task templates, source diversity, scoring criteria), expected_dataset_size, evaluation_plan, citations, confidence. Cathedral's own agent-trace data may be a primary source — name it explicitly if relevant."*

3. **"Benchmark drift brief"** — *"You are tracking benchmark stability for distillation work. Identify any changes to popular benchmarks (additions, deprecations, methodology changes) in the last 7 days that affect distillation quality measurement..."*

4. **"Cross-subnet handoff card"** — *"Given task class `<task_class>`, produce a Card whose `distillation_plan` is directly usable as an SN97 Distil miner submission. Match SN97's submission schema for `target_model`, `dataset_url`, `eval_protocol`."*

## Card schema extension — distillation-specific fields

```python
class DistillationPlan(BaseModel):
    teacher_model: str  # HuggingFace model ID
    teacher_benchmark: dict[str, float]  # benchmark -> score
    student_model_class: str  # arch + size, e.g. "llama-3.1-8b"
    student_target_metric: dict[str, float]  # benchmark -> target score
    dataset_specification: str
    expected_compute_hours: float | None = None  # H100-equivalents
    sn97_compatible: bool = False  # can be submitted as-is to SN97 Distil

class Card(BaseModel):
    # ... existing fields ...
    distillation_plan: DistillationPlan | None = None  # NEW — required for model-distillation cards
```

## Scoring rubric (this card type)

Distillation cards weight differently than regulatory cards:

- source_quality: 15% (papers + benchmarks; less rigid than regulatory primaries)
- maintenance: 10% (weekly cadence already low-pressure)
- freshness: 25% (model landscape moves fast — stale cards are useless)
- specificity: 25% (the plan must be concrete enough to execute)
- usefulness: 15% (sn97_compatible bonus)
- clarity: 10%

**New scorer**: `_distillation_plan_quality()` validates that the plan is concrete enough — e.g., teacher_model resolves on HuggingFace, benchmark scores are present, student_model_class is a real arch family, expected_compute_hours is a positive number.

## Cross-subnet bridge (SN97 Distil)

This is the operational dependency. The plan:

1. **DM SN97 owner this week.** Frame: "Cathedral is launching a distillation job category that maps to SN97 mining. Want to align on schema so our cards are directly usable as SN97 miner submissions."
2. **Schema alignment.** SN97's miner submission shape needs to match Cathedral's `DistillationPlan` output. Either we match theirs or they extend to match ours; bilateral.
3. **Bidirectional benefit:**
   - Cathedral cards → SN97 miner submissions (supply pipeline for SN97)
   - SN97 miner-result quality → Cathedral validator scoring signal (post-hoc — did the recommended distillation actually work?)
4. **Failure modes:**
   - SN97 doesn't agree on schema → we ship the cards anyway with `sn97_compatible=False`. Cards still useful as standalone intelligence.
   - SN97 disputes our scoring of distillation plans → bilateral rubric review, neither side gets veto.

## Adversarial considerations

- **Stale-paper spam**: agents that recommend a 2-year-old model as the "best" teacher fail freshness AND specificity.
- **Benchmark cherry-picking**: agents that report only the benchmark where their teacher wins should fail. Templates require ≥2 benchmarks.
- **Fictional plans**: agents that recommend non-existent HuggingFace models should fail validation. Validator pulls the HF model card during scoring — 404 = score=0.

## Open questions

1. **Does SN97 actually want this?** Hasn't been DM'd yet. Cathedral-side prep can proceed assuming yes; final go/no-go is bilateral.
2. **Compute estimation accuracy is hard.** Should `expected_compute_hours` be scored, or just recorded for retrospective analysis?
3. **Can we score the plan post-hoc?** I.e., did the plan actually produce a good distilled model when executed? This is a 30-90 day lag. Worth tracking but probably not in launch scoring.
4. **Cathedral's own agent traces as a dataset.** If Cathedral cards feed distillation, that's recursive — agents reasoning about distillation use traces of agents reasoning about regulation. Is that valuable or noisy? Worth A/B-ing once we have data.
5. **Licensing.** Some HF models are non-commercial. Should the card explicitly check license compatibility for the target use?

## Acceptance criteria

- [ ] `DistillationPlan` added to `src/cathedral/types.py`
- [ ] `model-distillation` registered in `src/cathedral/cards/registry.py`
- [ ] `card_definitions` row created with 4 templates + initial source pool
- [ ] `_distillation_plan_quality()` scorer added with HF API validation
- [ ] DM sent to SN97 Distil owner; schema alignment confirmed or noted as deferred
- [ ] First card lands on cathedral.computer leaderboard
- [ ] At least one card flagged `sn97_compatible=True`
- [ ] Adversarial test cases for stale-paper, benchmark-cherry-pick, fictional-model in `tests/v1/exploits/`

## References

- #89 — prompt-firing strategy
- [[20-research/training-subnet-history/distil-sn97-code-read]] (vault)
- Const's 2026-05-11 strategic-advice DM thread (vault: `30-people/peers/const`)
